src/pages/networks/panels/CreateLocalPeeringPanel.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  ActionButton,
+  Button,
+  ScrollableContainer,
+  SidePanel,
+  useNotify,
+  useToastNotification,
+} from "@canonical/react-components";
+import { useState, type FC } from "react";
+import usePanelParams from "util/usePanelParams";
+import * as Yup from "yup";
+import { useFormik } from "formik";
+import NotificationRow from "components/NotificationRow";
+import ResourceLink from "components/ResourceLink";
+import { queryKeys } from "util/queryKeys";
+import { useQueryClient } from "@tanstack/react-query";
+import { useCurrentProject } from "context/useCurrentProject";
+import type { LxdNetwork } from "types/network";
+import { createNetworkPeer } from "api/network-local-peering";
+import NetworkLocalPeeringForm from "../forms/NetworkLocalPeeringForm";
+import type { LocalPeeringFormValues } from "../forms/NetworkLocalPeeringForm";
+import { testDuplicateLocalPeeringName } from "util/networks";
+
+interface Props {
+  network: LxdNetwork;
+}
+
+const CreateLocalPeeringPanel: FC<Props> = ({ network }) => {
```

Comment:
This blocks submit in a valid state:

<img width="565" height="944" alt="image" src="https://github.com/user-attachments/assets/c9386ac3-98c0-47f1-afd3-4c05f8f79bed" />